### PR TITLE
Update JenkinsPipelineUnit to 1.7

### PR DESCRIPTION
**Pull request by nre-ableton**
Related to https://github.com/AbletonAppDev/devtools/issues/2999

### CODE REVIEW

**Comment by nre-ableton** [comment]
@ablbot rebase

**Comment by nre-ableton** [comment]
Unassigning @dgl-ableton since she is on holidays.

File: test/com/ableton/VirtualEnvTest.groovy [hunk 1]
```diff
@@ -103,60 +103,50 @@ class VirtualEnvTest extends BasePipelineTest {
 
   @Test
   void newObjectUnix() {
-    if (JenkinsMocks.isUnix()) {
```
Comment (mst-ableton) [inline test/com/ableton/VirtualEnvTest.groovy:106]: why are we removing the checks here?

Comment (nre-ableton) [reply test/com/ableton/VirtualEnvTest.groovy:106]: Previously we only ran these tests on the respective platform that they were tested on. Now we run all tests on each platform, mocking `isUnix`. To tell you the truth, I'm not sure why we did this before. See 8e0a0b7af0c6f099587a045ff6cc4ea87c4be7ee.

**Pull request merged**